File: ModelVB/DatabaseContext.vb
Imports System.Data.Entity

Public Class DatabaseContext
    Inherits DbContext

    Public Sub New(ByVal dbName As String)
        MyBase.New(dbName)
        Database.SetInitializer(New Initializer())
    End Sub

    Public Property Students As DbSet(Of Student)
    Public Property Teachers As DbSet(Of Teacher)
End Class

File: ModelCS/DatabaseContext.cs
﻿using System.Data.Entity;

namespace OOPRecords.Model
{
    public class DatabaseContext : DbContext
    {
        public DatabaseContext(string dbName) : base(dbName)
        {
            Database.SetInitializer(new Initializer());
        }

        public DbSet<Student> Students { get; set; }
        public DbSet<Teacher> Teachers { get; set; }
    }
}
